components/lib/crash/src/main/java/mozilla/components/lib/crash/service/SentryService.kt

```diff
@@ -57,6 +60,34 @@ class SentryService(
             client.sendMessage(createMessage(crash))
         }
     }
+
+    @Suppress("ComplexMethod")
```

Comment:
I like the way we can use when to covert from one enum to another.  Not sure if there's a better way to do this.

Reply:
You could break that down into extension methods, e.g. something like `internal fun CrashBreadcrumb.toSentryBreadcrumb()` and sub methods like `CrashBreadcrumb.Type.toSentryBreadcrumbType()`... but I think suppressing is valid option here too.

Reply:
Good idea.  I'll do it.  thanks,